app/src/main/java/com/example/lhi06/residue/AlbumReviewActivity.java

```diff
@@ -16,17 +22,34 @@
     @BindView(R.id.textview_album_name_review) TextView albumName;
     @BindView(R.id.textview_artist_name_review) TextView artistName;
     @BindView(R.id.imageview_album_art_review) ImageView albumArtwork;
+    @BindView(R.id.ratingBar) RatingBar ratingBar;
+    @BindView(R.id.reviewText) EditText reviewText;
+    @BindView(R.id.floatingbutton_savereview) FloatingActionButton saveReviewButton;
+
+    SaveReviews saveReviews;
+    List<AlbumReview> reviews;
 
     @Override
     protected void onCreate(@Nullable final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_album_review);
         ButterKnife.bind(this);
 
+        saveReviews.getInstance();
+        reviews = saveReviews.loadData(this);
+
         final Album album = getIntent().getExtras().getParcelable(FindAlbumActivity.ALBUM_EXTRA);
         albumName.setText(album.getCollectionName());
```

Comment:
To be on teh safe side you need a null check on `album` before using it as there you can't be sure the `getParcelable()` won't return null